core/src/main/java/io/fabric8/maven/core/util/FatJarDetector.java

```diff
@@ -13,7 +13,7 @@
  * implied.  See the License for the specific language governing
  * permissions and limitations under the License.
  */
-package io.fabric8.maven.generator.javaexec;
+package io.fabric8.maven.core.util;
```

Comment:
why is this change needed?

Reply:
It is indeed required. FatJarDetector was previously only being used by the `JavaExec` generator only. But now we're reusing this utility in `JibBuildService` as well. That's why its best to shift this Utility where it belongs better and can be re-used conveniently.